packages/application-shell/package.json

```diff
@@ -83,7 +83,7 @@
     "moment": "^2.24.0",
     "moment-timezone": "^0.5.26",
     "omit-empty-es": "1.0.3",
-    "perfume.js": "1.2.1",
+    "perfume.js": "2.1.2",
```

Comment:
Can’t update now.

Reply:
What do you mean? The TTI got removed, I suppose we should drop it as well...right? 🤔

Reply:
Downgraded: https://github.com/commercetools/merchant-center-application-kit/pull/1040/commits/98a71b1a60802d0c25db6b5ab5fc43b46ba350a7

We can open a separate PR.